xgcm/test/test_grid.py

```diff
@@ -640,4 +640,18 @@ def test_grid_ops(all_datasets):
                     assert da_cumsum.equals(da_cumsum_ax)
 
 
+# @pytest.mark.parametrize("func", ["interp", "diff", "max", "min"])
+@pytest.mark.parametrize("func", ["interp"])
+def test_multi_axis_input(all_datasets, func):
+    ds, periodic, expected = all_datasets
+    grid = Grid(ds)
+    axes = list(grid.axes.keys())
+    for varname in ["data_c", "data_g"]:
+        serial = ds[varname]
+        for axis in axes:
+            serial = getattr(grid, func)(serial, axis)
```

Comment:
I am also wondering if we can use something like this to generalize all the functions that act on a single axis (diff, interp, min, max, shift) and save ourselves some boilerplate code? Not necessarily important right now.